cmake/mpiexec.ursa

```diff
@@ -12,4 +12,4 @@ QOS=debug
 NP=$1
 shift
 
-srun -A $ACCOUNT -q $QOS -n $NP $@
+srun -A $ACCOUNT -q $QOS -t 0:01 -n $NP $@
```

Comment:
confirming, no more than 1 minute for all items that use this?

Reply:
Yes. Each individual test runs in under 10 seconds. Most run in under 1 second.  (There are 55 tests). If you don't specify the wall clock time, you are given a default of 5 minutes. One day I was getting slow turnaround from the queue. So, I set it to 1 minute. I don't know if it made a difference or not.